spec/text_normalizer/normalize_title_spec.rb

```diff
@@ -14,6 +14,10 @@
     it 'should reverse title parts when reverse is true' do
       expect(subject.normalize_title('', %w[title subtitle], '-', true)).to eq('subtitle-title')
     end
+
+    it 'should escape ampersand in title correctly' do
+      expect(subject.normalize_title(nil, %q(&&&), nil)).to eq("&amp;&amp;&amp;")
```

Comment:
Line is too long. [81/80]<br>Use `%q` only for strings that contain both single quotes and double quotes.